src/content/docs/style-guide/writing-docs/writer-workflow/process-changes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Proposing process or workflow changes
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Use this document to learn how to propose changes to the process or workflow in a collaborative manner.
+
+## Proposing a change
+
+Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.
```

Comment:
```suggestion
Do you have an idea you want to try out? It can be something big or something little or even something you haven't figured out how to solve yet. It can be a small suggestion such as using conventional commits, or a large-scale change such as automating PR reviews. If so, follow these steps so the team can collaborate on the issue or suggestion.
```

Reply:
Mostly I just wanted to try out the suggestion thing. Slick!